src/Psalm/Internal/Codebase/Functions.php

```diff
@@ -485,9 +485,13 @@ public function isCallMapFunctionPure(
 
             // bcmath
             'bcscale',
-            
+
             // json
             'json_last_error',
+
+            // opcache
+            'opcache_compile_file', 'opcache_get_configuration', 'opcache_get_status',
```

Comment:
I really dislike this alignment approach, as it makes identifying individual functions quite hard, but I kept consistent with the previous style.

Since no sorting was evident, I pushed it to the end of the block.

Meanwhile, is this supposed to be tested or meaningfully testable somehow?

Reply:
Not really, just as the callmap isn't meaningfully testable.